src/machine/aot/aot.x64.c

```diff
@@ -369,6 +369,14 @@ int aot_link(AotContext* context, size_t *szp)
   | and edx, CKB_VM_ASM_MEMORY_FLAG_WXORX_BIT
   | cmp edx, CKB_VM_ASM_MEMORY_FLAG_WRITABLE
   | jne >4
+  /*
+   * Set the page as dirty
+   */
+  | lea rdx, machine->flags
+  | movzx edx, byte [rdx+rcx]
+  | and edx, CKB_VM_ASM_MEMORY_FLAG_DIRTY
+  | lea rdx, machine->flags
```

Comment:
This line already changes the value of rdx, which means `dl` below is not a correct value.

Reply:
Yes, I wrote the wrong logic.